addons/loyalty/models/loyalty_program.py

```diff
@@ -50,7 +50,14 @@ class LoyaltyProgram(models.Model):
         ('next_order_coupons', 'Next Order Coupons')],
         default='promotion', required=True,
     )
-    date_to = fields.Date(string='Validity')
+    date_from = fields.Date(
+        string="Start Date",
+        help="The start date is included in the validity period of this program"
+    )
```

Comment:
Nitpicking

```suggestion
    date_from = fields.Date(
        string="Start Date",
        help="The start date is included in the validity period of this program",
    )
```
The comma here is so that if you add another line, you can put it after without having a 2 lines diff =]
And here, we are in the exception we talk about, but you can still see both, especially in loyalty since we didn't clean it yet, but this would also be accepted:

```suggestion
    date_from = fields.Date(
        string="Start Date",
        help="The start date is included in the validity period of this program")
```